src/components/ProjectSideMenu/ProjectSideMenuLayout.tsx

```diff
@@ -0,0 +1,55 @@
+import { ProjectSideMenuContext } from "@/contexts/projectSideMenu"
+import { createStyles, Stack } from "@mantine/core"
+import React, { useContext } from "react"
+
+type ProjectSideMenuLayoutProps = {
+    topButtons: React.ReactNode
+    bottomButtons: React.ReactNode
+}
+
+const useStyles = createStyles((theme) => ({
+    container: {
+        position: "relative",
+        height: "calc(100vh - 60px)",
+        background: theme.colors.dark[7],
+        left: 0,
+        top: 0,
+        paddingTop: 2,
+    },
+}))
+
+export const ProjectSideMenuLayout: React.FC<ProjectSideMenuLayoutProps> = ({ topButtons, bottomButtons }) => {
```

Comment:
1. Component and file name `ProjectSideMenuLayout` -> `SideMenu`
2. Move to `src/components`
3. no context dependency
4. `topButtons` -> `children`
5. `bottomButtons` -> `extra?`

Reply:
fix linter warnings